onadata/apps/viewer/tests/test_tasks.py

```diff
@@ -38,3 +40,11 @@ def test_create_async(self):
             self.assertTrue(export.id)
             self.assertIn("username", options)
             self.assertEquals(options.get("id_string"), self.xform.id_string)
+
+    def test_check_pending_exports(self):
+        result = check_pending_exports.delay()
+        self.assertTrue(result)
```

Comment:
We should probably check that the function deletes exports and not that a task is created.